contrib/arm/inffast_chunk.c

```diff
@@ -0,0 +1,311 @@
+/* inffast.c -- fast decoding
+ * Copyright (C) 1995-2017 Mark Adler
+ * For conditions of distribution and use, see copyright notice in zlib.h
```

Comment:
I wonder if we should point clearly that this is a modded inffast.c (i.e. by adding the respective Copyright).